data-raw/unique_medicines_per_patient_df.R

```diff
@@ -0,0 +1,52 @@
+# Load library
+library(magrittr)
+
+# Set up connection to DALP
+con <- nhsbsaR::con_nhsbsa(database = "DALP")
+
+# Create a lazy table from the care home FACT table
+fact_db <- dplyr::tbl(
+  src = con,
+  from = dbplyr::sql("SELECT * FROM DALL_REF.INT615_ITEM_LEVEL_BASE")
+)
+
+# Create a laxy table to return DRUG ATTRIBUTE information from CDR_DRUG_BNF_DIM
+drug_db <- dplyr::tbl(
+  src = con,
+  from = dbplyr::sql(
+    "SELECT * FROM SB_DIM.CDR_DRUG_BNF_DIM
+  WHERE 1=1
+  AND YEAR_MONTH in
+  (202004, 202005, 202006, 202007, 202008, 202009,
+  202010, 202011, 202012, 202101, 202102, 202103)"
+  )
+)
```

Comment:
Think we can do a more `{dbplyr}` method to do this?

I propose either:

* Leaving it out and just relying on the join to `fact_db` to limit the period
* Check out some of the scripts in branch 6 but there are some where we load in `year_month_dim` and do the inner join to that...

I think the first way is a bit cleaner?